package.json

```diff
@@ -22,32 +22,43 @@
     "ghpages:build": "rimraf ghpages && yarn run prepare && vite build",
     "ghpages:publish": "yarn run ghpages:prepare -e $(git branch --show-current)",
     "ghpages:prepare": "yarn run ghpages:build && touch ghpages/.nojekyll && yarn run ghpages:generate",
-    "ghpages:generate": "gh-pages -t -d ghpages -m \"Build for $(git log --pretty=format:\"%h %an %ai %s\" -n1) [skip ci]\""
+    "ghpages:generate": "gh-pages -t -d ghpages -m \"Build for $(git log --pretty=format:\"%h %an %ai %s\" -n1) [skip ci]\"",
+    "publish:alpha": "npm run test && npm run prepare && npm version prerelease --preid alpha && npm publish --tag alpha"
```

Comment:
Q: `npm version` by default creates a new tagged commit w/ the version change, right?

I'm wondering whether we actually want that behavior for these alpha tags, since we generally try to avoid committing our alphas. I believe one can skip the commit by specifying the `--no-git-tag-version` option on the `npm version` command, if desired.

Reply:
Thanks @latonv. 
> we generally try to avoid committing our alphas
Is this documented? There may be some vagueness around the term "commit".  Happy not to tag, but need to `git commit` in order to publish the alpha. Please expand, thanks again.

Reply:
the `--no` is severely ill-documented in https://docs.npmjs.com/cli/v8/using-npm/config

But my concern is that I understand what you mean by "commit"

Reply:
Added `--no-git-tag-version` option on the `npm version` command